### PR TITLE
Update Screenshots and UG

### LINKED ISSUE #176: Docs-UG: Update screenshots
As we changed the UI, screenshots of the old interface should be updated throughout the documentation.

### PR DESCRIPTION
**Pull request by bgopi23**
Resolves #176

### CODE REVIEW

**Comment by bgopi23** [comment]
@dillontkh @jjchee77 do u know why its failing the markdown check? i cant seem to find the issue

**Comment by RyanNgWH** [comment]
@bgopi23 it seems to be looking for a link `#editing-a-client` that does not exist in your version of the doc but is on the master branch. 

Could you try merging with master and ensuring that the link works?

**Comment by bgopi23** [comment]
@RyanNgWH the section does exist tho, also merged with master and its still failing

**Comment by RyanNgWH** [comment]
@bgopi23 okay seems like the link was indeed broken. Could you try replacing the link in line 212 with `(#editing-a-client--edit)`

**Comment by RyanNgWH** [comment]
I also noticed some discrepancies in the header format for the features section. Some of them have a space before the colon (e.g line 130) while others do not (e.g line 140). 

Could you help to standardize them?

**Comment by bgopi23** [comment]
@RyanNgWH ok thx

File: docs/UserGuide.md [hunk 1]
```diff
@@ -150,6 +150,8 @@ A client can have 0 or more tags.
 Examples:
 * `add n/John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01 nt/john from school`
 * `add n/Betsy Crowe t/friend e/betsycrowe@hotmail.com a/Newgate Prison p/1234567 nt/likes donuts t/criminal`
+
+Format: `add`
```
Comment (dillontkh) [inline docs/UserGuide.md:154]: Is this supposed to be here?

**Review by dillontkh — COMMENTED** [review]
LGTM, just 1 part that seems like a typo

**Review by dillontkh — APPROVED** [review]
LGTM!

**Review by jjchee77 — APPROVED** [review]
LGTM!

**Pull request merged**